kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -1679,6 +1684,7 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         menu.findItem(R.id.unmuteGroup).setVisible(false);
         menu.findItem(R.id.deleteConversation).setVisible(false);
         menu.findItem(R.id.logout).setVisible(alCustomizationSettings.isLogoutOptionFromConversation());
+        menu.findItem(R.id.rate_conversation).setVisible(alCustomizationSettings.isRatingFromConversationEnabled());
```

Comment:
We have one more option to enable/disable CSAT. It is available in dashboard. We are storing it in App Setting Preference.
use this check here as well: `themeHelper.isCollectFeedback()`

Reply:
Done as requested